tests/php/Control/DirectorTest.php

```diff
@@ -744,40 +749,19 @@ public function testForceSSLandForceWWW()
         $this->assertEquals(301, $response->getStatusCode());
     }
 
-    /**
-     * Set url to redirect to
-     *
-     * @var string
-     */
-    protected $expectedRedirect = null;
-
     /**
      * Expects this test to throw a HTTPResponse_Exception with the given redirect
-     *
-     * @param string $url
      */
-    protected function expectExceptionRedirect($url)
-    {
-        $this->expectedRedirect = $url;
-    }
-
-    protected function runTest()
```

Comment:
runTest() is now a final method so it cannot be overriden. I've reimplemented this functionality by changing it the testing methodology to `withExpectedExceptionRedirect()` instead